source/Panel.h

```diff
@@ -79,7 +79,7 @@ class Panel {
 
 protected:
 	// Only override the ones you need; the default action is to return false.
-	virtual bool KeyDown(SDL_Keycode key, Uint16 mod, const Command &command, bool isNewPress);
+	virtual bool KeyDown(int32_t key, const Command &command, bool isNewPress);
```

Comment:
Previously, you could specify a mod when calling KeyDown inside of a panel; but it will now always use the mod from the user's keyboard. This doesn't break any current code that I can see, but that's by luck.

Reply:
I mean deliberatly specifying wrong mods sounds like a bad idea?

Reply:
It's to force a specific keybind to activate. We have several cases where we have both mouse and keyboard options for certain activities; in some cases, it's easier to have the mouse event generate a "fake" key event on the same component to deduplicate the code. One such example is LoadPanel.

Reply:
:confused: I am not sure that is a good argument

Reply:
both the key and the mouse should trigger a third separate structure in a nice structure

Reply:
Feel free to open a refactor.

Reply:
mate my time is already short as is